src/fdb5/api/FDB.h

```diff
@@ -72,73 +76,200 @@ class FDB {
+    /// @return DataHandle for reading the requested data from
     eckit::DataHandle* retrieve(const metkit::mars::MarsRequest& request);
 
+    // TODO(kkratz): Provide doc!
     ListIterator inspect(const metkit::mars::MarsRequest& request);
 
+    /// List data present at the archive and which can be retrieved.
+    /// @param request FDBToolRequest stating which data should be queried
+    /// @param deduplicate bool whether the returned iterator should ignore duplicates
+    /// @param length Size of the data to archive with the given @p key
+    /// @return ListIterator for iterating over the set of found items
     ListIterator list(const FDBToolRequest& request, bool deduplicate = false, int level = 3);
 
+    /// Dump the structural content of the FDB
+    /// In particular, in the TOC formulation, enumerate the different entries
+    /// in the Table of Contents (including INIT and CLEAR entries).
+    /// The dump will include information identifying the data files that are
+    /// referenced, and the "Axes" which describe the maximum possible extent of
+    /// the data that is contained in the database.
+    /// @param request
+    /// @param simple
+    /// @return DumpIterator for iterating over the set of found items
     DumpIterator dump(const FDBToolRequest& request, bool simple = false);
 
-    /// TODO: Is this function superfluous given the control() function?
+    // TODO(kkratz): Provide doc!
     StatusIterator status(const FDBToolRequest& request);
 
+    /// Wipe data from the database.
+    /// Deletes FDB databases and the data therein contained. Uses the passed
+    /// request to identify the database to delete. This is equivalent to a
+    /// UNIX rm command.
+    /// This tool deletes either whole databases, or whole indexes within databases
+    /// @param request FDBToolRequest stating which data should be queried
+    /// @param doit flag for committing to the wipe (default is dry-run)
+    /// @param porcelain flag print only a list of files to be deleted / that are deleted
+    /// @param unsafeWipeAll flag for omitting all security checks and force a wipe
+    /// @return WipeIterator for iterating over the set of wiped items
     WipeIterator wipe(const FDBToolRequest& request, bool doit = false, bool porcelain = false,
                       bool unsafeWipeAll = false);
 
+    /// Move content of one FDB database.
+    /// This locks the source database, make it possible to create a second
+    /// database in another root, duplicates all data. Source data are not automatically removed.
+    /// @param request a fdb tool request for the data which should be move
+    /// @param dest destination uri to which the data should be moved
+    /// @return MoveIterator for iterating over the set of found items
     MoveIterator move(const FDBToolRequest& request, const eckit::URI& dest);
 
+    /// Remove duplicate data from the database.
```

Comment:
General comment, that these need some 'vertical whitespace' (I guess blank /// lines). This is a bit of a 'block' of text.

Reply:
It's good to have a clear one-line summary of the routine, and then detail laid out below if needed.